src/legacy/core_plugins/kbn_doc_views/public/views/table/table_helper.test.ts

```diff
@@ -0,0 +1,177 @@
+    const actual = convertAngularHtml('<span ng-non-bindable>Good morning!</span>');
+    expect(actual).toMatchInlineSnapshot(`"Good morning!"`);
+  });
+  it(`converts html containing <mark> for usage in react`, () => {
+    const actual = convertAngularHtml(
+      '<span ng-non-bindable>Good <mark>morning</mark>dear <mark>reviewer</mark>!</span>'
+    );
+    expect(actual).toMatchInlineSnapshot(`
+                        <React.Fragment>
+                          Good 
+                          <span>
+                            <mark>
+                              morning
+                            </mark>
+                            dear 
+                          </span>
+                          <span>
+                            <mark>
+                              reviewer
+                            </mark>
+                            !
+                          </span>
+                        </React.Fragment>
+                `);
+  });
+});
+
+describe('arrayContainsObjects', () => {
+  it(`returns false for an array of primitives`, () => {
+    const actual = arrayContainsObjects(['test', 'test']);
+    expect(actual).toBeFalsy();
+  });
+
+  it(`returns true for an array of objects`, () => {
+    const actual = arrayContainsObjects([{}, {}]);
+    expect(actual).toBeTruthy();
+  });
+
+  it(`returns true for an array of objects and primitves`, () => {
+    const actual = arrayContainsObjects([{}, 'sdf']);
+    expect(actual).toBeTruthy();
+  });
+
+  it(`returns false for an array of null values`, () => {
+    const actual = arrayContainsObjects([null, null]);
+    expect(actual).toBeFalsy();
+  });
+
+  it(`returns false if no array is given`, () => {
+    const actual = arrayContainsObjects([null, null]);
+    expect(actual).toBeFalsy();
+  });
+});
+
+describe('formatValue', () => {
+  it(`formats an array of objects`, () => {
+    const actual = formatValue([{ test: '123' }, ''], '');
+    expect(actual).toMatchInlineSnapshot(`
+      "{
+        \\"test\\": \\"123\\"
+      }
+      \\"\\""
+    `);
+  });
+  it(`formats an array of primitives`, () => {
+    const actual = formatValue(['test1', 'test2'], '');
+    expect(actual).toMatchInlineSnapshot(`"test1, test2"`);
+  });
+  it(`formats an object`, () => {
+    const actual = formatValue({ test: 1 }, '');
+    expect(actual).toMatchInlineSnapshot(`
+      "{
+        \\"test\\": 1
+      }"
+    `);
+  });
+  it(`formats an angular formatted string `, () => {
+    const actual = formatValue(
+      '',
+      '<span ng-non-bindable>Good <mark>morning</mark>dear <mark>reviewer</mark>!</span>'
+    );
+    expect(actual).toMatchInlineSnapshot(`
+      <React.Fragment>
+        Good 
+        <span>
+          <mark>
+            morning
+          </mark>
+          dear 
+        </span>
+        <span>
+          <mark>
+            reviewer
+          </mark>
+          !
+        </span>
+      </React.Fragment>
+    `);
+  });
+});
```

Comment:
FYI @flash1293 Additionally to the changes you suggested, I've added Jest tests for the helper functions